app/components/find_interface/courses/financial_support/scholarship_and_bursary_component/view.rb

```diff
@@ -7,19 +7,39 @@ class ScholarshipAndBursaryComponent::View < ViewComponent::Base
         delegate :scholarship_amount,
           :bursary_amount,
           :has_early_career_payments?,
-          :subject_name, to: :course
+          to: :course
 
         def initialize(course)
           super
           @course = course
         end
 
         def scholarship_body
-          I18n.t("find.scholarships.#{subject_name.downcase}.body", default: nil)
+          I18n.t("find.scholarships.#{subject_with_scholarship}.body", default: nil)
         end
 
         def scholarship_url
-          I18n.t("find.scholarships.#{subject_name.downcase}.url", default: nil)
+          I18n.t("find.scholarships.#{subject_with_scholarship}.url", default: nil)
+        end
+
+      private
+
+        SUBJECT_WITH_SCHOLARSHIPS = [
+          %w[F1 chemistry],
```

Comment:
Feels like we should put these into a config somewhere to re-reference. One for the tech debt stuff maybe

Reply:
Yeah... thro there is a wider tech debt with the components as well